product_category_pricelist_button/models/product_category.py

```diff
@@ -0,0 +1,35 @@
+# © 2024 Solvos Consultoría Informática (<http://www.solvos.es>)
+# License LGPL-3.0 (https://www.gnu.org/licenses/lgpl-3.0.html)
+
+from odoo import models, fields
+
+class ProductCategory(models.Model):
+    _inherit = 'product.category'
+
+    pricelist_count = fields.Integer(compute='_compute_pricelist_count', store=True,
+        help="The number of pricelists associated with products under this category."
+    )
+
+    def _compute_pricelist_count(self):
```

Comment:
Once `pricelist_count` is now stored, an `@api.depends` clause should be added, could you make the required modifications?